recipe/meta.yaml

```diff
@@ -48,37 +47,38 @@ requirements:
     - icu 56.*  # [linux]
   run:
     - python
+    - setuptools
     - numpy x.x
     - cycler >=0.10
     - python-dateutil
     - freetype 2.6.*
     - pytz
     - pyparsing
-#    - py2cairo  # [linux and py2k]
+    #- py2cairo  # [linux and py2k]
     - libpng >=1.6.28,<1.7
     - pyqt 4.11.*  # [not osx]
     - tk 8.5.*  # [linux]
     - functools32  # [py2k]
     - subprocess32  # [py2k and unix]
     - icu 56.*  # [linux]
-    - setuptools
+    - tornado
```

Comment:
`tornado` is a run-time dependency but was listed at build-time only and, for some reason, fooled `conda-smithy` to drop all Python 3 from the build matrix. I guess that we don't really need to add it here since `tornado` seems to be used only for `matplotlib`+`jupyter` features and `jupyter` does install `tornado`. Is that assessment correct?